doc/chips/11.rst

```diff
@@ -115,27 +106,28 @@ We did not incorporate this functionality because it adds complexity to
 how initialization occurs.
 
 
-Parent could complete initialization after child's initialization is complete
-*****************************************************************************
-
-We considered allowing the parent class to perform final initialization
-actions after the child class(es) have established their initialization
-invariants.
+Parent could have final control over the newly-created object
+-------------------------------------------------------------
 
-An example of such an action is adding the new object to a shared list.
+We considered giving the parent class the ability to exercise final control
+over the newly-created object. An example where this is important
```

Comment:
I would extend this first sentence to explicitly mention how this would be accomplished: "We considered giving the parent class the ability to exercise final control over the newly-created object via a 'lastly' block."  Or something along those lines.